tools/cldr-code/src/main/java/org/unicode/cldr/util/Organization.java

```diff
@@ -56,10 +57,24 @@ public enum Organization {
     wikimedia("Wikimedia Foundation"),
     wod_nko("WOD N’ko", "World Organization for the Development of N’ko", "WODN"),
     yahoo("Yahoo"),
-
-    // To be removed.
     ;
 
+    /**
+     * Get a list of the TC Organizations
+     * @return
+     */
+    public static Set<Organization> getTCOrgs() {
+        return EnumSet.of(google, apple, microsoft);
```

Comment:
Make a private final static TC_ORGS = ImmutableSet.copyOf(EnumSet.of(google, apple, microsoft));

No reason to recreate a mutable set each time this is called.

Reply:
I think I did this originally because i forgot to make TC_ORGS static and this wasn't allowed. Fixed now.